core/arch/arm/plat-hikey/conf.mk

```diff
@@ -1,7 +1,7 @@
 include core/arch/$(ARCH)/plat-$(PLATFORM)/platform_flags.mk
 
 ifeq ($(CFG_ARM64_core),y)
-CFG_WITH_LPAE := y
+CFG_WITH_LPAE ?= y
```

Comment:
It's quite bad if this is overridden. Perhaps we should have some macro to set variables that really need a specific value?